### PR TITLE
Fix Travis CI build.

### PR DESCRIPTION
**Pull request by artem-zinnatullin**
Cmon travis, just trigger the build damn it :/

### CODE REVIEW

**Comment by artem-zinnatullin** [comment]
Lol, why u check CLA on forks?

**Comment by artem-zinnatullin** [comment]
Though I am talking to myself in a lonely fork… who am i to judge a bot doing its work…

**Comment by artem-zinnatullin** [comment]
Oh damn, it's not my fork, apologies to everyone! (I specifically clicked "New PR" on my own fork to test PR before opening it against upstream)

**Comment by cwoodwar6** [comment]
Also bump the buck version to bring in the fix for https://github.com/facebook/buck/issues/1510 since it was causing failures on travis.

File: .travis.yml [hunk 1]
```diff
@@ -1,25 +1,27 @@
...
-
 before_script:
   - export SKIP_OKBUCK=true
   - export EXTRA_OKBUCK_ARGS="--quiet --stacktrace"
-  - mkdir -p $ANDROID_HOME/licenses
-  - echo $android_sdk_license > $ANDROID_HOME/licenses/android-sdk-license
-  - echo $android_sdk_preview_license > $ANDROID_HOME/licenses/android-sdk-preview-license
+  - export ANDROID_HOME="$HOME"/android-sdk
+  - mkdir -p "$ANDROID_HOME"
+  - export ANDROID_SDK_FILE_NAME=sdk-tools-darwin-3859397.zip
```
Comment (kageiit) [inline .travis.yml]: why darwin? travis runs on linux right?

**Comment by kageiit** [comment]
would be great if you can bump the buckversion too

File: .travis.yml (hunk 1 above)
Comment (artem-zinnatullin) [reply .travis.yml]: ah, facepalm, now I'm curious how it works in other projects where I did same thing… I guess Android tooling is more or less crossplatform now, that's how you check that lol

**Comment by kageiit** [comment]
Will update buck version separately

**Comment by artem-zinnatullin** [comment]
Ahh, I haven't have time to fix absence of `android-wait-for-emulator`, will post separate PR

**Pull request merged**